Gradata/src/gradata/cloud/materializer.py

```
@@ -0,0 +1,389 @@
+            winning_device_id=str(data.get("device_id") or ""),
+        )
+    return result
+
+
+__all__ = [
+    "CONFLICT_THRESHOLD",
+    "Conflict",
+    "MaterializedRule",
+    "MaterializeResult",
+    "materialize",
+]
```

Ruff (0.15.10)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)